apps/qubit/modules/user/actions/clipboardLoadAction.class.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+class UserClipboardLoadAction extends DefaultEditAction
+{
+  // Arrays not allowed in class constants
+  public static
+    $NAMES = array(
+      'password',
+      'mode');
+
+  protected function earlyExecute()
+  {
+
+  }
```

Comment:
What is this for? There is no earlyExecute in the parent class to overwrite, right?

Reply:
Yeah, I'm doing a bit of "cargo cult"-ing here. IIRC without this added to the class a fatal error gets generated and I noticed that a bunch of other classes do the same thing with this method.